src/views/Predictions/hooks/useIsRefundable.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react'
```

Comment:
I'd rather put the hooks under `src/hooks/predictions`, like it's done for the ifo.This folder is a View folder

Reply:
This file is gone. However I don't think it should go in the hooks folder. It is not meant to be used anywhere but predictions.